### DISCUSSION TITLE
总在最后一步报错，怎么办，关闭了防火墙了，用的是腾讯云的服务器

### DISCUSSION THREAD

**Discussion by z2labplus** [Jinnrry/PMail#118] [General]
46

**Comment by Jinnrry** [comment]
check that a DNS record exists for this domain

DNS记录配错了，或者没生效，检查你的DNS配置。配置如果没问题的话等一个小时再试，可能是DNS缓存没刷新

**Reply by Jinnrry** [reply]
有啥问题你直接这里说就行了。你用nslookup之类的命令先排查下dns生效了没，没生效的话就等会

**Reply by Jinnrry** [reply]
你这个解析还差一个根路径的解析吧，再配一个 A记录，主机名 "@"，指向你服务器

**Comment by z2labplus** [comment]


**Reply by Jinnrry** [reply]
DNS problem: NXDOMAIN looking up A for pmail.jidukp.com - check that a DNS record exists for this domain; 

你的dns记录里面也没pmail这个记录呀

**Reply by Jinnrry** [reply]
嗯
